modules/git/bootstrap.Makefile

```diff
@@ -34,8 +34,8 @@ else
   export GIT_IS_BRANCH := 1
 endif
 
-export GIT_URL ?= $(shell git remote get-url origin 2>/dev/null)
```

Comment:
I noticed this uses raw `git` instead of `$(GIT)` ... I'm leaving it as is, but if folks think that's a mistake, I can change it. @mcalhoun last touched this code, AFAIK... so maybe they have ideas?

Reply:
It seems like a mistake to not use `$(GIT)`

Reply:
Yea, let's fix this, then merge. We can always do follow up PRs. But this affects too many to ignore.